postcss.config.js

```diff
@@ -0,0 +1,12 @@
+const postcssLogical = require('postcss-logical');
+const postcssDirPseudoClass = require('postcss-dir-pseudo-class');
+const postcssFlexGapPolyfill = require('flex-gap-polyfill');
+
+module.exports = {
+    syntax: 'postcss-scss',
+	plugins: [
+        postcssLogical({preserve: true}),
+        postcssDirPseudoClass(),
+        postcssFlexGapPolyfill()
+	]
```

Comment:
hm, the formatting is off here, but not in VSCode, not sure why Prettier didn't format it, will look into it